### PR TITLE
Create getallBlooksonLocker

### PR DESCRIPTION
**Pull request by DavidKitKat**

### CODE REVIEW

**Comment by DavidKitKat** [comment]
it only works on chromebook

**Comment by DavidKitKat** [comment]
https://user-images.githubusercontent.com/106819340/172095159-e3cf1617-23ab-4cd5-b128-25ea2b9395f9.mp4

**Comment by DavidKitKat** [comment]
https://www.youtube.com/watch?v=mHrXtuZhLA0

File: global/get-all-blooks-on-locker [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+HOW IT WORKS:
+Triple click the code
```
[the remark this answers is not in the document]
Comment (glixxzzy) [reply global/get-all-blooks-on-locker]: it doesn't work.

[the remark this answers is not in the document]
Comment (DavidKitKat) [reply global/get-all-blooks-on-locker]: i think it only works on a specific blooket account cause it works for me

https://user-images.githubusercontent.com/106819340/172296656-c6b3c738-e0e1-458a-b89a-26ac89608f85.mp4

**Pull request closed without merging**